### PR TITLE
fixed code snippet and text errors

### PR DESCRIPTION
**Pull request by Lukeghenco**
Updated the code snippet to be cleaner and more readable (added constructor, initial state and a checked attribute to the input tag), I also cleaned up some of the written text and styling.

@aturkewi @PeterBell

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,19 +27,33 @@ In React, having a component that needs to have state is also a clue that it sho
 Here's an example of a React component whose state changes whenever its child element (a checkbox) is clicked.
 
 ```
+import React, { Component } from 'react'
```
Comment (aturkewi) [inline README.md:30]: If you're going to only import `Component` from React, then you need to update the syntax on line 32 to be `class myCheckBox extends Component {`

File: README.md [hunk 2]
```diff
@@ -27,19 +27,33 @@ In React, having a component that needs to have state is also a clue that it sho
 Here's an example of a React component whose state changes whenever its child element (a checkbox) is clicked.
 
 ```
```
Comment (aturkewi) [inline README.md]: Can you add `js` right after the three backticks (no space) so we get some JS formatting?

File: README.md [hunk 3]
```diff
@@ -50,11 +64,16 @@ When a user clicks the checkbox, this triggers the `handleCheckBoxClick` functio
...
+State in Redux is similar to state in React but at a larger scale. While state in React is focused on an individual component and its children, **Redux holds the state for our whole application**. Redux state refers to both the data that our application needs to use in our layout **PLUS** the logic around how that data is displayed. For example, let's say that we're using our checkbox React component in an app used at the Flatiron School that keeps track of student progress. We need to keep track of the students in the class and figure out if any of them haven't gotten approval from an instructor, for their app idea.
```
Comment (aturkewi) [inline README.md]: Given that this curriculum is supposed to be for online and in person, how do you feel about changing the last sentence to something like: "We need to keep track of the students that have graduated." or something more generic like that?

**Review by aturkewi — CHANGES_REQUESTED** [review]
Made a couple of comments. Let me know what you think!

File: README.md (hunk 3 above)
Comment (Lukeghenco) [reply README.md]: fixed

**Review by Lukeghenco — COMMENTED** [review]
finished

**Pull request merged**